### DISCUSSION TITLE
Local compound data base search is not combined with the spectra match

### DISCUSSION THREAD

**Discussion by IzaFurlani** [mzmine/mzmine#2225] [General]
Dear team,

I am processing GCMS data, and I want to combine a spectra library with an in-house database that contains some information about standards that we acquire together (as a QC and a useful annotation source). 
From the mzwizard, I am including both libraries, and everything looks fine without errors. However, once it is processed, I only can see the matches related to the spectra library and not from the in-house database. I would like to combine both!

Is there anything that I might have forgotten to do? 

![mzmine DB](https://github.com/user-attachments/assets/c87d6ce0-c73f-4925-8067-f7b08aee2c43)

**Comment by ansgarkorf** [comment]
Hi,
local compound database search for GC/EI is difficult, because most of the time you dont observe the precursor mz. The mz in the feature list is from the most intense fragment. You would need to adjust your csv file to look for exactly these mz values.

**Reply by IzaFurlani** [reply]
Hi,

Thank you for your reply.

Basically, the way I've prepared this list was separated based on mz, rt, name, and formula (check the image attached). 
What do you mean about adjusting to look exactly these values? It would be selecting the option of  "use precursor m/z --> as listed in database"?
Could you check if it is correct? Thank you very much
![mzmine DB_v02](https://github.com/user-attachments/assets/301d9792-fc1b-49f2-80e5-1595529053e0)
